### PR TITLE
added initContainers to helm template

### PR DESCRIPTION
**Pull request by bhataprameya**
added initContainers to helm template.

### CODE REVIEW

**Comment by difabion** [comment]
Thanks for the contribution.  Could you also add the new option to the list in the README for your PRs?  I'll bump the chart version once all of them have been added.

**Comment by bhataprameya** [comment]
Done. Added README

**Comment by bhataprameya** [comment]
Added README  updates for all my  PR's

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ For more information on configuring the HTTP component, refer to the [Benthos HT
 | http.tls.enabled                              | Enables TLS for all Benthos endpoints            | false              |
 | http.tls.secretName                           | `kubernetes.io/tls` secret name                  | ""                 |
 | watch                                         | Enables watch mode                               | false              |
+| initContainers                                | Add any custom inti container                    | []                 |
```
Comment (difabion) [inline README.md:71]: typo - `init`

**Review by difabion — APPROVED** [review]

**Comment by difabion** [comment]
Approved the lot, thanks.  I'll get these merged and released.

**Comment by bhataprameya** [comment]
@difabion any timeline on these approved PRS releases?

**Pull request merged**